ui/public/i18n/en/translation.json

```
@@ -18,7 +18,11 @@
     "mount": "Mount",
     "no_services": "No services",
     "no_images": "No images",
-    "no_volumes": "No volumes"
+    "no_volumes": "No volumes",
+    "not_configured": "Not configured",
+    "sogo_webapp": "SOGo webmail",
+    "open_webapp": "Open app",
```

```suggestion
    "open_webapp": "Open SOGo",
```
Let's make it uniform to Nextcloud and other webapps